CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ cmake_minimum_required(VERSION 2.8.7)
 set (randomx_sources
 src/aes_hash.cpp
 src/argon2_ref.c
+src/argon2_sse3.c
```

Comment:
Since SSE3 exists and is different from SSSE3, please use "ssse3" or "SSSE3" everywhere instead, to avoid confusion.